integration-tests/repositories/factories/simple-product-category-factory.ts

```diff
@@ -0,0 +1,12 @@
+import { Connection } from "typeorm"
```

Comment:
question: Now that we have started using factories across multiple projects, should we also centralize them in a dedicated place? Wdyt?

Reply:
I'd prefer that. Is there any reason why they're independent packages and not just one?

Either case, I'll create an issue for centralising the factories.

Reply:
> Is there any reason why they're independent packages and not just one

I am not sure I follow. What packages?

Reply:
@olivermrbl the `integration-test/(api|plugins|repositories)` folders are created as independent private packages.